examples/composite_purchase_order.sample

```diff
@@ -1,16 +1,15 @@
 {
   "purchase_order": {
     "id": null,
-    "comments": "History Connect 2017",
     "created": "2008-03-15T03:53:00Z",
```

Comment:
What date format should we use? "2008-03-15T03:53:00Z" or "2008-03-15T03:53:00.000+0000"

Reply:
I looked around the various repos and unfortunately there isn't much consistency across the board...  I see four formats used:
2008-03-15T03:53:00Z
2008-03-15T03:53:00.000Z
2008-03-15T03:53:00.0000+0000
2008-03-15T03:53:00.000+00:00

I think the best we can do is be consistent within a given module.  I know this came up before with the mock data and the 2008-03-15T03:53:00.000+0000 format was recommended at that time, but I think the 2008-03-15T03:53:00.000Z format is probably the preferred way to go since that's what javascript (and therefore the UI) uses by default (e.g. Date.toJson(), etc).

If the chosen date format continues to cause problems with the mock data and the mod-orders unit tests, the problem will be addressed in the unit tests.

FWIW, these example json files need a lot of work.  I did just the bare minimum to get them to validate against the schemas, so there are many missing fields, and in some cases the data doesn't make sense... (things like eresource indicates it's an electronic resource, but then only quantity_physical is populated), etc.

Reply:
Thank you for full explanation